### DISCUSSION TITLE
Date format in v3.24.0 is kinda long

### DISCUSSION THREAD

**Discussion by PeteMacca** [OpenTracksApp/OpenTracks#1092] [General]
Hi, love the app! Latest release has changed the date format, making it very long. Bug or design? See attached image. Pete
![Screenshot_20220114-163547](https://user-images.githubusercontent.com/97723277/149463294-f7a2d949-d303-4161-9530-00c3db1320ac.png)

**Comment by dennisguse** [comment]
Yes, there was a change on this code as we now save the offset (due to timezone) as well.
We can reduce the formatting here I think.

**Comment by rgmf** [comment]
Maybe it's not necessary for naming tracks. Users can already see offset on track's details.

**Comment by dennisguse** [comment]
Fixed in https://github.com/OpenTracksApp/OpenTracks/issues/1093
